qa/tasks/vstart_runner.py

```diff
@@ -1201,11 +1201,22 @@ def __del__(self):
         if test_path:
             shutil.rmtree(test_path)
 
+
+class LogRotate():
+    def __init__(self):
+        self.conf_file_path = os.path.join(os.getcwd(), 'logrotate.conf')
+        self.state_file_path = os.path.join(os.getcwd(), 'logrotate.state')
+
+    def run_logrotate(self):
+        remote.run(args=['logrotate', self.conf_file_path, '-s',
```

Comment:
```suggestion
        remote.run(args=['logrotate', '-f', self.conf_file_path, '-s',
```

So that each test run starts fresh for easier debugging.

Reply:
Modified.